configuration.json

```diff
@@ -2,14 +2,111 @@
   "videoRecord": false,
   "videoDir": "build/cify/videos/",
   "capabilities": {
-    "android": {
-      "capability": "android"
-    },
-    "browser": {
-      "capability": "chrome"
-    },
-    "ios": {
-      "capability": "iphone"
-    }
+    "browser": [
+      {
+        "capability": "chrome",
+        "capabilityId": "ChromeMac"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      },
+      {
+        "capability": "chrome"
+      }
+    ],
+    "android": [
+      {
+        "capability": "android",
```

Comment:
The same here. I think there is no need to have multiple android capabilities and capability id

Reply:
Yep